Running on euler with following parameters : 
User : atabin
Model : unet
Data folder : /cluster/work/igp_psr/ai4good/group-1b/data/
Saving directory : /cluster/scratch/atabin/unet/
Checkpoint path : /cluster/scratch/atabin/checkpoints/unet/
usage: run.py [-h] [--data_dir DATA_DIR] [--checkpoint_path CHECKPOINT_PATH]
              [--save_dir SAVE_DIR]
              {cgnet,upernet}
run.py: error: argument model: invalid choice: 'unet' (choose from 'cgnet', 'upernet')
Done!
